### DISCUSSION TITLE
Overselling/Ignore stock value

### DISCUSSION THREAD

**Discussion by Einarsson** [saleor/saleor#12991] [General]
Hello!

 I might be dumb but I've searched far and long for the ability to ignore the stock values when adding a product to cart.

I simply wan't to be able to add to cart even if there are no products in stock.

Thanks for your time. :-)

Cheers!

**Comment by timuric** [comment]
Hey @Einarsson you can disable stock tracking in settings of variants

<img width="1728" alt="image" src="https://github.com/saleor/saleor/assets/4006792/56c2b370-6a9b-4f50-a372-449ba40a1dd9">

<img width="882" alt="image" src="https://github.com/saleor/saleor/assets/4006792/3335979d-68ac-4c04-bfd6-414a75a778a3">

**Reply by Einarsson** [reply]
Hey, thanks for input.

I did that, unchecked the box, still getting:

```
[
  {
    code: 'INSUFFICIENT_STOCK',
    message: 'Could not add items Standard. Only 0 remaining in stock.',
    field: 'quantity'
  }
]

dashboard v3.13.3
core v3.13.17
```

If it matters I'm running https://github.com/saleor/nextjs-commerce

**Reply by Einarsson** [reply]
<img width="776" alt="Screenshot 2023-06-01 at 14 36 29" src="https://github.com/saleor/saleor/assets/526098/b8041a38-49ac-438b-9e28-be2743d8f178">
<img width="1393" alt="Screenshot 2023-06-01 at 14 37 15" src="https://github.com/saleor/saleor/assets/526098/bd61ee9a-7099-4884-b693-afa0dbeb13e8">

**Reply by Einarsson** [reply]
I get the same error even if I click the box AND add 100 quantity to it.

**Reply by Einarsson** [reply]
@timuric Don't know how to reopen an issue but I'll tag you and hope you see it. :-)

**Reply by Einarsson** [reply]
Strange, I recreated teh database and used testdata and now it works. Possibly some data got corrupt or something in our instance.
